cms/djangoapps/contentstore/management/commands/tests/test_backfill_orgs_and_org_courses.py

```diff
@@ -55,9 +56,17 @@ def test_end_to_end(self):
         # org_B: already existing, but has no content.
         add_organization({"short_name": "org_B", "name": "Org B"})
 
-        # org_C: has a couple courses; should be created.
+        # org_C: has a few courses; should be created.
         CourseOverviewFactory(org="org_C", run="1")
         CourseOverviewFactory(org="org_C", run="2")
+        # Include an Old Mongo Modulestore -style deprecated course key.
+        # This can be safely removed when Old Mongo Modulestore support is
+        # removed.
+        CourseOverviewFactory(
+            id=CourseLocator.from_string("org_C/toy/3"),
+            org="org_C",
+            run="3",
+        )
```

Comment:
I wrote this before adding the bug fix, and confirmed that it failed with the expected error message.